contents/handbook/people/team-structure/pipeline/objectives.mdx

```diff
@@ -0,0 +1,69 @@
+- Objective: Develop Posthog CDP into a solution capable of replacing other CDPs for data-warehousing categories. Increase % of customers using “data out” apps from X to Y
+    - Key result:
+        - Make the most important data in and out apps work reliably
+            - (Potentially) Consolidate into 1 repo
+            - Opinionated way of writing tests (maybe end-to-end), structuring the apps
+            - Separate out filters
+        - Team to fill in
+    - Ideas
+        - CDP style UI
+            - Why? To make the CDP product feel like a standalone product similar to other CDP solutions
+        - Live view of data or event testing
+    - Why?
+        - By removing the need for other CDPs, Posthog can save the customers money and potentially create a new revenue line. 
+        - Historically we’ve fixed things iteratively. We want to consider this as a product.
+- Objective: Improve the performance of these areas
+    - Key results:
+        - Re-write how we do historical exports
+            - Able to cope with lots of volume
+            - Resource limiting the apps
+            - Why?
+                - We promised it to our largest customer
+        - (Proposed lower priority) Performance - improve ingestion performance
+            - Why?
+            - High volume customers e.g. a very large open source devtool customer couldn’t use our ingestion pipeline and so had to insert directly into clickhouse
+
+
+## Old roadmap
```

Comment:
Q: Do we care about objectives or do we want a road map? :thinking: 

The two are in my opinion very different things as one prescribes "achieve X as a company/team" and the other "build Y and Y exactly". Of course one informs the other but a task queue doesn't help with decision-making beyond tasks listed, leads to frustration as unexpected obstacles occur and is a recipe for quickly going out of sync with people in different teams/execs.